src/ui/navigation.R

```diff
@@ -20,15 +20,12 @@ panelWithTitle <- function(title, content) {
 enableNavLinks <- function(input, output, spectrumFilesState, modelRunState, surveyAndProgramData) {
     enableTabWhen("Upload survey data", function() { spectrumFilesState$anyDataSets() })
     enableTabWhen("Upload programmatic data", function() { spectrumFilesState$anyDataSets() })
-    enableTabWhen("Review input data", function() { spectrumFilesState$anyDataSets() &&
```

Comment:
No need for the review tab to be locked down to when program data is valid, in fact you want it to be enabled so that you see the validation message.